assistantv1/assistant_v1_integration_test.go

```diff
@@ -38,12 +38,11 @@ func init() {
 	if err == nil {
 		service, serviceErr = assistantv1.
 			NewAssistantV1(&assistantv1.AssistantV1Options{
-				URL:      os.Getenv("ASSISTANT_GO_SDK_URL"),
-				Version:  "2018-09-20",
-				Authenticator: &core.BasicAuthenticator{
-						Username: os.Getenv("ASSISTANT_GO_SDK_USERNAME"),
-						Password: os.Getenv("ASSISTANT_GO_SDK_PASSWORD"),
-                },
+				URL:     os.Getenv("ASSISTANT_URL"),
```

Comment:
Just curious... why not use the built-in external config support to set service-level properties (URL) and the authenticator?

Reply:
Originally, that was because of the confusion with differences in service names. But I think Ill update the env variable with the service names with `conversation` and `watson-vision-combined`

Reply:
Actually, you will probably need to wait on that until your team decides to use the latest version of the generator that has the service factory feature implemented.   Part of that feature is ensuring that all the languages use the same service name for a particular service.   No more assistant vs conversation inconsistencies :)